msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

```diff
@@ -364,9 +388,30 @@ The default value of the property `isChatResourceSpecificConsentEnabled` is base
 <br>
 </details>
 
+<br>
+<details>
+
+<summary><b>Configure user owner consent settings for RSC in a user using the Graph APIs</b></summary>
+
+You can enable or disable RSC for user using Graph API. The property `isUserPersonalScopeResourceSpecificConsentEnabled` in [teamsAppSettings](/graph/api/teamsappsettings-update#example-1-enable-installation-of-apps-that-require-resource-specific-consent-in-chats-meetings) governs whether user RSC is enabled in the tenant.
+
+:::image type="content" source="../../assets/images/rsc/graph-rsc-user-configuration.PNG" alt-text="The screenshot shows the Graph RSC user configuration.":::
+
+The default value of the property `isUserPersonalScopeResourceSpecificConsentEnabled` is based on whether [user consent settings](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-portal) is turned on or off in the tenant when RSC for user is first used. The default value is defined either when:
+
+* [TeamsAppSettings](/graph/api/teamsappsettings-get) are retrieved for the first time.
+* Teams app with RSC permissions is installed in for a user.
```

Comment:
Teams app with RSC permissions is installed for a user

Reply:
Updated.